docs/datadog_checks_dev.cli.rst

```diff
@@ -40,3 +40,33 @@ Commands
 .. click:: datadog_checks.dev.tooling.cli:ddev
    :prog: ddev
    :show-nested:
+
+Command Tab Completion
+^^^^^^^^^^^^^^^^^^^^^^
+
+Many of the `ddev` commands have tab-completion enabled.
+There are a couple of ways to activate this for your system. The first is to
+execute the following in your shell:
+
+.. code-block:: bash
+
+    $ eval "$(_DDEV_COMPLETE=source ddev)"
+
+or for `zsh`:
+
+.. code-block:: bash
+
+    $ eval "$(_DDEV_COMPLETE=source_zsh ddev)"
+
+Hitting `tab` at any point in your `ddev` command will autocomplete with available
+options.  Note that this will only work for `integrations-core` repo, and
+not any of the extended repos `ddev` can support.
+
+For a more permenant solution, you can add the lines to either your `.bashrc` or `.zhsrc`
+files respectively, but this may introduce a bit of a lag on startup.  As an alternative,
+run the following command then load the resulting file into your startup script:
```

Comment:
What's the difference between a) dumping the lines into a script and executing the script on terminal startup (i.e. in `.zshrc), and b) adding the `eval ...` command to the `.zshrc`? I assume the result is to run the generated lines either way, so both ways have the same startup lag caveat?

Reply:
Based on my initial testing, there was a little more lag associated with the `eval`, since it needs to execute `ddev` to generate the equivalent completion file vs having that file already available for sourcing.

Testing it just now, the difference seemed negligible, but that could be due to caching.

Reply:
> since it needs to execute ddev to generate the equivalent completion file

Ah I see, makes sense now. :-)